### PR TITLE
Added coverage for LDAP auth source

### PR DESCRIPTION
**Pull request by ntkathole**
"""
pytest tests/foreman/ui/test_ldapauthsource.py::LDAPAuthSourceTestCase::test_positive_create_withad_org_and_loc
========================== test session starts ===================================
collected 5 items
tests/foreman/ui/test_ldapauthsource.py .

================================= 1 passed ===================================
"""

### CODE REVIEW

File: robottelo/ui/base.py [hunk 1]
```diff
@@ -831,3 +837,31 @@ def perform_entity_action(self, action_name):
         """
         self.click(common_locators['kt_select_action_dropdown'])
         self.click(common_locators['select_action'] % action_name)
+
+    def _configure_orgs(self, orgs, org_select):
```
Comment (oshtaier) [inline robottelo/ui/base.py]: Hm. That is really bad idea to put these methods to the most base class of framework. Move them to `robottelo/ui/ldapauthsource.py` or wherever you need them

**Comment by ntkathole** [comment]
@oshtaier updated changes...thanks for review

**Comment by oshtaier** [comment]
Pending one more run results

**Comment by ntkathole** [comment]
@oshtaier and @kbidarkar Updated test results:

```

pytest tests/foreman/ui/test_ldapauthsource.py::LDAPAuthSourceTestCase::test_positive_create_withad_org_and_loc
============================= test session starts ==============================
collected 5 items
tests/foreman/ui/test_ldapauthsource.py .

========================== 1 passed  ========================
```

**Pull request merged**